spec/index.bs

```diff
@@ -6346,10 +6333,16 @@ dictionary GPUShaderModuleDescriptor
 
     : <dfn>sourceMap</dfn>
     ::
-        If defined MAY be interpreted as a source-map-v3 format.
+        If defined, **may** be interpreted in the [[!SourceMap]] v3 format.
+
+        If an implementation supports this option but is unable to process the provided value,
+        it should show a developer-visible warning but must not produce any application-observable
+        error.
```

Comment:
I was wondering about this behavior just the other day. Thanks for the clarifying text!